data/org.eclipse.birt.data.aggregation/.settings/org.eclipse.jdt.core.prefs

```diff
@@ -70,7 +70,7 @@ org.eclipse.jdt.core.compiler.problem.unusedParameterWhenOverridingConcrete=disa
 org.eclipse.jdt.core.compiler.problem.unusedPrivateMember=warning
 org.eclipse.jdt.core.compiler.problem.varargsArgumentNeedCast=warning
 org.eclipse.jdt.core.compiler.release=enabled
-org.eclipse.jdt.core.compiler.source=11
+org.eclipse.jdt.core.compiler.source=16
```

Comment:
It seems odd to set it to 16.

Reply:
Yes. It was done by Eclipse quick-fix. I switched to 17 manually afterwards and missed one line.